android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/DebugMenuModule.java

```diff
@@ -93,26 +95,19 @@ public void simulatePositiveDiagnosis(Callback callback) {
     }
 
     @ReactMethod
-    public void detectExposuresNow(Callback callback) {
+    public void detectExposuresNow(Promise promise) {
         ExposureNotificationClientWrapper.get(getReactApplicationContext())
                 .isEnabled().addOnSuccessListener(
                 enabled -> {
                     if (enabled) {
-                        ProvideDiagnosisKeysWorker.scheduleDailyProvideDiagnosisKeys(getReactApplicationContext());
-                        callback.invoke(null, CallbackMessages.DEBUG_DETECT_EXPOSURES_SUCCESS);
+                        WorkManager workManager = WorkManager.getInstance(getReactApplicationContext());
+                        workManager.enqueue(new OneTimeWorkRequest.Builder(ProvideDiagnosisKeysWorker.class).build());
```

Comment:
Trigger a one time request of scheduling the worker

Reply:
Just confirming, is another job still running once per day to download keys and check exposures?

Reply:
Yes, the 24 hours job will still run once we call `client.start()`